.config/nvim/ftplugin/java.lua

```diff
@@ -1,142 +1,159 @@
 -- JDTLS (Java LSP) configuration
-local jdtls = require('jdtls')
-local project_name = vim.fn.fnamemodify(vim.fn.getcwd(), ':p:h:t')
-local workspace_dir = vim.env.HOME .. '/jdtls-workspace/' .. project_name
+local home = vim.env.HOME -- Get the home directory
+
+local jdtls = require("jdtls")
+local project_name = vim.fn.fnamemodify(vim.fn.getcwd(), ":p:h:t")
+local workspace_dir = home .. "/jdtls-workspace/" .. project_name
+
+-- Determine OS
+-- Use this option to avoid explicitly adding manually everytime you switch OS e.g "config_linux", "config_mac"
```

Comment:
I apologize ahead of time, but my code reviews are very nit-picky.  Can you remove the 2 comment lines on 9 and 10, and make the variable a local variable called os or system_os or something like that? and please include a windows option?

Reply:
Done!